fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -118,7 +116,7 @@
 public class FedoraVersioningIT extends AbstractResourceIT {
 
     public static final DateTimeFormatter MEMENTO_DATETIME_ID_FORMATTER =
-        DateTimeFormatter.ofPattern("yyyyMMddHHmmss").withZone(ZoneId.of("GMT"));
+            DateTimeFormatter.ofPattern("yyyyMMddHHmmss").withZone(ZoneId.of("GMT"));
```

Comment:
Quite a bit of unrelated formatting changes in this file. It would make the review (and merge conflicts) easier without such updates.